### PR TITLE
Update for Node 1.4 (Polaris)

### PR DESCRIPTION
**Pull request by andreibancioiu**
- Upgrade setup for Node 1.4 (Polaris).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,50 +1,59 @@
...
+RUN git clone https://github.com/multiversx/mx-chain-go --branch=$(cat /workspace/mx-chain-mainnet-config/binaryVersion | sed 's/tags\///') --single-branch mx-chain-go-mainnet
+RUN git clone https://github.com/multiversx/mx-chain-proxy-go.git --branch=${PROXY_TESTNET_TAG} --single-branch --depth=1 mx-chain-proxy-go-testnet
 RUN git clone https://github.com/multiversx/mx-chain-proxy-go.git --branch=${PROXY_DEVNET_TAG} --single-branch --depth=1 mx-chain-proxy-go-devnet
 RUN git clone https://github.com/multiversx/mx-chain-proxy-go.git --branch=${PROXY_MAINNET_TAG} --single-branch --depth=1 mx-chain-proxy-go-mainnet
 
-# Build node, proxy and keygenerator:
+# Build node and proxy
+WORKDIR /go/mx-chain-go-testnet/cmd/node
+RUN go build -i -v -ldflags="-X main.appVersion=$(git describe --tags --long --dirty --always)"
+RUN cp /go/pkg/mod/github.com/multiversx/mx-chain-vm-v1_4-go@$(cat /go/mx-chain-go-testnet/go.mod | grep mx-chain-vm-v1_4-go | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /go/mx-chain-go-testnet/cmd/node/libwasmer_linux_amd64.so
```
Comment (iulianpascalau) [inline Dockerfile:29]: please replace with:
```
RUN cp /go/pkg/mod/github.com/multiversx/$(cat /go/mx-chain-go/go.mod | grep mx-chain-vm-v | sort -n | tail -n -1| awk -F '/' '{print$3}'| sed 's/ /@/g')/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so
```

File: Dockerfile [hunk 2]
```diff
@@ -1,50 +1,59 @@
...
 
-ARG CONFIG_DEVNET_TAG=D1.3.50.0-hf01
-ARG CONFIG_MAINNET_TAG=v1.3.50.0
-ARG PROXY_DEVNET_TAG=v1.1.27
-ARG PROXY_MAINNET_TAG=v1.1.27
+ARG CONFIG_TESTNET_TAG=T1.4.8.0
+ARG CONFIG_DEVNET_TAG=D1.4.8.0
+ARG CONFIG_MAINNET_TAG=v1.4.8.0
+ARG PROXY_TESTNET_TAG=v1.1.33
+ARG PROXY_DEVNET_TAG=v1.1.33
```
Comment (iulianpascalau) [inline Dockerfile:7]: according to the latest discussion we now have `v1.1.34` :)

File: Dockerfile [hunk 3]
```diff
@@ -1,50 +1,59 @@
...
-# Build node, proxy and keygenerator:
+# Build node and proxy
+WORKDIR /go/mx-chain-go-testnet/cmd/node
+RUN go build -i -v -ldflags="-X main.appVersion=$(git describe --tags --long --dirty --always)"
+RUN cp /go/pkg/mod/github.com/multiversx/mx-chain-vm-v1_4-go@$(cat /go/mx-chain-go-testnet/go.mod | grep mx-chain-vm-v1_4-go | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /go/mx-chain-go-testnet/cmd/node/libwasmer_linux_amd64.so
+
 WORKDIR /go/mx-chain-go-devnet/cmd/node
 RUN go build -i -v -ldflags="-X main.appVersion=$(git describe --tags --long --dirty --always)"
-RUN cp /go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@$(cat /go/mx-chain-go-devnet/go.mod | grep arwen-wasm-vm | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so
+RUN cp /go/pkg/mod/github.com/multiversx/mx-chain-vm-v1_4-go@$(cat /go/mx-chain-go-devnet/go.mod | grep mx-chain-vm-v1_4-go | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /go/mx-chain-go-devnet/cmd/node/libwasmer_linux_amd64.so
```
Comment (iulianpascalau) [inline Dockerfile:33]:
```
RUN cp /go/pkg/mod/github.com/multiversx/$(cat /go/mx-chain-go/go.mod | grep mx-chain-vm-v | sort -n | tail -n -1| awk -F '/' '{print$3}'| sed 's/ /@/g')/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so
```

File: Dockerfile [hunk 4]
```diff
@@ -1,50 +1,59 @@
...
 
 WORKDIR /go/mx-chain-go-mainnet/cmd/node
 RUN go build -i -v -ldflags="-X main.appVersion=$(git describe --tags --long --dirty --always)"
-RUN cp /go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@$(cat /go/mx-chain-go-mainnet/go.mod | grep arwen-wasm-vm | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so
+RUN cp /go/pkg/mod/github.com/multiversx/mx-chain-vm-v1_4-go@$(cat /go/mx-chain-go-mainnet/go.mod | grep mx-chain-vm-v1_4-go | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /go/mx-chain-go-mainnet/cmd/node/libwasmer_linux_amd64.so
```
Comment (iulianpascalau) [inline Dockerfile:37]:
```
RUN cp /go/pkg/mod/github.com/multiversx/$(cat /go/mx-chain-go/go.mod | grep mx-chain-vm-v | sort -n | tail -n -1| awk -F '/' '{print$3}'| sed 's/ /@/g')/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so
```

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**